server/JsDbg.Gdb/JsDbg.py

```diff
@@ -344,7 +344,11 @@ def GetSymbolsInStackFrame(instructionAddress, stackAddress, frameAddress):
         frame = frame.older()
 
     if frame:
-        block = frame.block()
+        try:
+            block = frame.block()
+        except:
+            # We are probably missing symbols for this frame.
```

Comment:
Will we end up showing "??" for this case?

Reply:
No, for this case we'd just show no local variables.

Reply:
Ok, thanks for the clarification.